node/kafkaproducer/mock_MessageProducer.go

```diff
@@ -1,8 +1,10 @@
 // Code generated by mockery v1.0.0. DO NOT EDIT.
 package kafkaproducer
 
-import kafka "github.com/confluentinc/confluent-kafka-go/kafka"
-import mock "github.com/stretchr/testify/mock"
+import (
```

Comment:
i'm not sure we should fix this one; the code is autogenerated by mockery, and when we regenerate it the fixes will be lost

Reply:
Right on. The DO NOT EDIT slid right past me apparently.